pkg/services/ngalert/api/api_contact_points.go

```diff
@@ -0,0 +1,71 @@
+package api
+
+import (
+	"errors"
+	"net/http"
+
+	"github.com/grafana/grafana/pkg/api/response"
+	api "github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/services/ngalert/services"
+	"github.com/grafana/grafana/pkg/web"
+)
+
+type ContactPointServer struct {
+	service services.ContactPointService
+}
+
+func (s *ContactPointServer) RouteGetContactPoint(c *api.ReqContext) response.Response {
+	if !c.HasUserRole(api.ROLE_VIEWER) {
+		return ErrResp(http.StatusForbidden, errors.New("permission denied"), "")
+	}
+	templates, err := s.service.GetContactPoints(c.OrgId)
+	if err != nil {
+		return ErrResp(http.StatusInternalServerError, err, "")
+	}
+	return response.JSON(http.StatusOK, templates)
+}
+
+func (s *ContactPointServer) RouteCreateContactPoint(c *api.ReqContext) response.Response {
+	if !c.HasUserRole(api.ROLE_EDITOR) {
+		return ErrResp(http.StatusForbidden, errors.New("permission denied"), "")
+	}
+	contactPoint := services.EmbeddedContactPoint{}
+	if err := web.Bind(c.Req, &contactPoint); err != nil {
+		return response.Error(http.StatusBadRequest, "bad request data", err)
+	}
+	contactPoint, err := s.service.CreateContactPoint(c.OrgId, contactPoint)
```

Comment:
`OrgID` is a domain field for a contact point; that is to say, a contact point always exists in an org.

Should we consider adding orgID as a field on the contact point struct rather than passing them around as a pair? It makes the domain object more complete. We can always just not serialize the field at the API layer if we want to continue not returning it in the response